src/user/user.service.ts

```diff
@@ -0,0 +1,58 @@
+import { Injectable } from '@nestjs/common';
+import { InjectRepository } from '@nestjs/typeorm';
+import { UserEntity } from './user.entity';
+import { CreateUserDto } from './dto/create-user.dto';
+import { Repository } from 'typeorm';
+import * as bcrypt from 'bcrypt';
+
+@Injectable()
+export class UsersService {
+  constructor(
+    @InjectRepository(UserEntity)
+    private userRepository: Repository<UserEntity>,
+  ) {}
+
+  async findOneBy(email: string): Promise<UserEntity | undefined> {
```

Comment:
(thought): The `findOneBy` naming convention suggestions you are passing a field to find a user by. For example, I would assume I would use it with `findOneBy('email', 'taylor@lamarsoftware.io');`